File: FusionAndroid/app/src/main/java/fr/intechinfo/fusionandroid/ContactsList.java
package fr.intechinfo.fusionandroid;

import java.util.List;

public class ContactsList {
    List<Contact> Contacts;

    public List<Contact> GetContact(){
        return this.Contacts;
    }

    public void SetContact(List<Contact> contact) {
        this.Contacts = contact;
    }
}

File: FusionAndroid/app/src/main/java/fr/intechinfo/fusionandroid/ContactsList.kt
package fr.intechinfo.fusionandroid

class ContactsList {
    internal var Contacts: List<Contact>? = null

    fun GetContact(): List<Contact>? {
        return this.Contacts
    }

    fun SetContact(contact: List<Contact>) {
        this.Contacts = contact
    }
}
